cilium-cli/api/api.go

```diff
@@ -74,7 +74,7 @@ var _ Hooks = &NopHooks{}
 func (*NopHooks) AddSysdumpFlags(*pflag.FlagSet)                                  {}
 func (*NopHooks) AddSysdumpTasks(*sysdump.Collector) error                        { return nil }
 func (*NopHooks) AddConnectivityTestFlags(*pflag.FlagSet)                         {}
-func (*NopHooks) AddConnectivityTests(*check.ConnectivityTest) error              { return nil }
+func (*NopHooks) AddConnectivityTests([]*check.ConnectivityTest) error            { return nil }
```

Comment:
Can we turn this into a variadic variable instead? That we can pass either a single connectivity test or multiple without having to initialize a slice variable.